### DISCUSSION TITLE
MAUI Blazor Hybrid iOS Fail

### DISCUSSION THREAD

**Discussion by TimurSuyarov** [dotnet/maui#13073] [Q&A]
Have a problem with balzor hybrid app to deploy iOS platform. When I wonna deploy `default` MAUI Blazor Hybrid app to iOS, it does not  show pages after splash image. `Visual Studio 17.4`, `.NET 6`, `Xcode 14.2`

**Comment by akhanalcs** [comment]
Does this help?
https://github.com/dotnet/maui/issues/13094
